src/TerminalFighter/TextArea/TextArea.h

```diff
@@ -0,0 +1,21 @@
+#include <string>
+
+#include <SDL_ttf.h>
+
+namespace{
+    const SDL_Color COLOR_WHITE = {255, 255, 255};
+    const SDL_Color COLOR_GRAY = {100, 100, 100};
+}
+
+class TextArea {
```

Comment:
This class will probably be a dependency for a test one day. It should have an interface super class.